bridges/snowbridge/pallets/inbound-queue-v2/src/test.rs

```diff
@@ -331,7 +331,7 @@ fn zero_reward_does_not_register_reward() {
 			Message {
 				nonce: 0,
 				assets: vec![],
-				xcm: XcmPayload::Raw(vec![]),
+				payload: Payload::Raw(vec![]),
```

Comment:
I don't think this should be renamed - the whole message is the payload, and this part relates specifically to XCM.